store/postgres/asset_repository_test.go

```diff
@@ -127,6 +130,104 @@ func (r *AssetRepositoryTestSuite) insertRecord() (assets []asset.Asset) {
 	return assets
 }
 
+func (r *AssetRepositoryTestSuite) TestBuildFilterQuery() {
+	r.builder = sq.Select(`a.test as test`)
+
+	r.Run("should not return all assets limited by default size", func() {
```

Comment:
actually you can put these tests into a [table driven test](https://dev.to/boncheff/table-driven-unit-tests-in-go-407b) to simplify it. You can check on of the tests in columbus repo.